.github/workflows/ci.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Install hyperopt optional dependencies (Python 3.11 only)
         if: ${{ matrix.python-version == '3.11' }} 
         run: |
-          python -m pip install ".[hpopt]"
+          python -m pip install ".[hpopt]" optuna
```

Comment:
I think maybe we can provide two possible installation for hpopt, one comes with hyperopt and the other optuna, so that the user can use `pip install ".[hpopt-hyperopt]"` or `pip install ".[hpopt-optuna]"` when they install the hpopt module. What do you think? It doesn't have to be `hpopt-hyperopt` and `hpopt-optuna`, but something along those lines.

Reply:
I think putting everything into `hpopt` should be fine (i.e., `hpopt = ["ray[tune]", "hyperopt", "optuna"]`). The only difference between `hpopt-hyperopt` and `hpopt-optuna` is whether to install `hyperopt` or `optuna`. If users prefer not to install the additional dependency of either one, they can manually install the packages they want using pip and I don't think we need to provide the additional installation choices for this.